sample_files/.travis.yml

```diff
@@ -27,4 +30,4 @@ script:
   - travis_run_tests
 
 after_success:
-  coveralls
```

Comment:
As we add MQT in the PATH, we could easily shadow `coveralls` indeed. And then, call directly`/usr/local/bin/coveralls` (path to check on travis). 

But it is safe to do so? Can we assume that the `coveralls` path won't change in travis builds? Also, an inconvenience is that it would add a layer of noise and technical debt.

What are the other opinions here?